source/_patterns/04-pages/homepage/homepage.twig

```diff
@@ -0,0 +1 @@
+{% include 'templates-single-column-grid' with {content: body} only %}
```

Comment:
The grid currently removes margin collapsing, but will be fixed when #54 is merged.

Reply:
Just spotted that #63 doesn't actually contain anything. Thoughts on doing the list here too given this pattern shouldn't be used on its own? /cc @davidcmoulton

Reply:
I agree @thewilkybarkid. If a teaser will only ever be used in a list, even in a list of only one item, then the atomic pattern unit is the list of teasers. (We could always break it out later if a future requirement arises.)

Reply:
This needs updating (as `templates-single-column-grid` was removed).

We'll need a `listing-grid`(?).

Reply:
> This needs updating (as `templates-single-column-grid` was removed).

I've updated it just to bring it inline with other pages but we may need more work done here?